packages/angular_devkit/build_optimizer/src/build-optimizer/build-optimizer.ts

```diff
@@ -16,20 +16,29 @@ import { getWrapEnumsTransformer, testWrapEnums } from '../transforms/wrap-enums
 
 
 const whitelistedAngularModules = [
-  /(\\|\/)node_modules(\\|\/)@angular(\\|\/)animations(\\|\/)/,
-  /(\\|\/)node_modules(\\|\/)@angular(\\|\/)common(\\|\/)/,
-  /(\\|\/)node_modules(\\|\/)@angular(\\|\/)compiler(\\|\/)/,
-  /(\\|\/)node_modules(\\|\/)@angular(\\|\/)core(\\|\/)/,
-  /(\\|\/)node_modules(\\|\/)@angular(\\|\/)forms(\\|\/)/,
-  /(\\|\/)node_modules(\\|\/)@angular(\\|\/)http(\\|\/)/,
-  /(\\|\/)node_modules(\\|\/)@angular(\\|\/)platform-browser-dynamic(\\|\/)/,
-  /(\\|\/)node_modules(\\|\/)@angular(\\|\/)platform-browser(\\|\/)/,
-  /(\\|\/)node_modules(\\|\/)@angular(\\|\/)platform-webworker-dynamic(\\|\/)/,
-  /(\\|\/)node_modules(\\|\/)@angular(\\|\/)platform-webworker(\\|\/)/,
-  /(\\|\/)node_modules(\\|\/)@angular(\\|\/)router(\\|\/)/,
-  /(\\|\/)node_modules(\\|\/)@angular(\\|\/)upgrade(\\|\/)/,
-  /(\\|\/)node_modules(\\|\/)@angular(\\|\/)material(\\|\/)/,
-  /(\\|\/)node_modules(\\|\/)@angular(\\|\/)cdk(\\|\/)/,
+  /[\\\/]node_modules[\\\/]@angular[\\\/]animations[\\\/]/,
+  /[\\\/]node_modules[\\\/]@angular[\\\/]common[\\\/]/,
+  /[\\\/]node_modules[\\\/]@angular[\\\/]compiler[\\\/]/,
+  /[\\\/]node_modules[\\\/]@angular[\\\/]core[\\\/]/,
+  /[\\\/]node_modules[\\\/]@angular[\\\/]forms[\\\/]/,
+  /[\\\/]node_modules[\\\/]@angular[\\\/]http[\\\/]/,
+  /[\\\/]node_modules[\\\/]@angular[\\\/]platform-browser-dynamic[\\\/]/,
+  /[\\\/]node_modules[\\\/]@angular[\\\/]platform-browser[\\\/]/,
+  /[\\\/]node_modules[\\\/]@angular[\\\/]platform-webworker-dynamic[\\\/]/,
+  /[\\\/]node_modules[\\\/]@angular[\\\/]platform-webworker[\\\/]/,
+  /[\\\/]node_modules[\\\/]@angular[\\\/]router[\\\/]/,
+  /[\\\/]node_modules[\\\/]@angular[\\\/]upgrade[\\\/]/,
+  /[\\\/]node_modules[\\\/]@angular[\\\/]material[\\\/]/,
+  /[\\\/]node_modules[\\\/]@angular[\\\/]cdk[\\\/]/,
+];
+
+const es5AngularModules = [
+  // Angular 4 packaging format has .es5.js as the extension.
+  /.es5.js$/, // Angular 4
```

Comment:
Probably doesn't matter, but (just to be safe) the `.` should be escaped (to avoid matching `foo.noes5.js` :grin:):

```js
/\.es5\.js$/
```

Reply:
Fixed